bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/types/StateDescription.java

```diff
@@ -98,7 +100,7 @@ public String getPattern() {
      *
      * @return true, if the state can only be read but not written
      */
-    public boolean isReadOnly() {
+    public @Nullable Boolean isReadOnly() {
```

Comment:
As there is now a third possible value (`null`) that could be returned, we should change the JavaDoc and explain what is the intention of `null`.